packages/sdk/src/identity/identityConfig.ts

```diff
@@ -0,0 +1,77 @@
+import { ECDSA_SECP256K1_EVM, ML_DSA_87, SigningUtil } from '@streamr/utils'
+import { KeyPairIdentityConfig, EthereumProviderIdentityConfig, StrictStreamrClientConfig, CustomIdentityConfig } from '../Config'
+import { EthereumKeyPairIdentity } from './EthereumKeyPairIdentity'
+import { EthereumProviderIdentity } from './EthereumProviderIdentity'
+import { Identity } from './Identity'
+import { MLDSAKeyPairIdentity } from './MLDSAKeyPairIdentity'
+import { SignatureType } from '@streamr/trackerless-network'
+
+/**
+ * This is where config keyTypes are connected to Identity implementations.
+ * 
+ * How to configure new Identity types:
+ * 1. Add a new SignatureType entry to NetworkRpc.proto in network package
+ * 2. Add the needed SigningUtil to signingUtils.ts
+ * 3. Create the Identity implementation itself (eg. extend KeyPairIdentity)
+ * 4. Wire everything together below
+ */
+export const validKeyTypeValues = ['ECDSA_SECP256K1_EVM', 'ML_DSA_87'] as const
+export const identityConfig: Record<KeyType, {
+    fromConfig: (config: Pick<StrictStreamrClientConfig, 'auth'>) => Identity
```

Comment:
Remove the internal method from public API

```suggestion
    /** @internal */
    fromConfig: (config: Pick<StrictStreamrClientConfig, 'auth'>) => Identity
```